lib/cirro_io_v2/request_clients/jwt.rb

```diff
@@ -14,6 +14,7 @@ def initialize(base_url:, private_key:, client_id:)
         @connection = Faraday.new(url: base_url) do |conn|
           conn.request :json
           conn.response :json
+          conn.adapter :net_http
```

Comment:
🤔 why net_http?

Reply:
Without it I'm getting
<img width="1215" alt="Screenshot 2022-08-05 at 17 56 50" src="https://user-images.githubusercontent.com/20345554/183092405-9d4bf06d-d701-4fd6-8df1-1ced4a624fe1.png">
 error.

Reply:
is it because of JsonAPI maybe? I don't remember any errors like that. Let me try it locally as well.

Reply:
I understand, it's strange. Because, as I know, Faraday uses net_http adapter by default.